plugins/quickbooks-online/src/src/Facade/QuickBooksFacade.php

```diff
@@ -236,7 +236,8 @@ public function exportInvoices(): void
             return;
         }
 
-        foreach ($this->ucrmApi->query('invoices?direction=ASC') as $ucrmInvoice) {
+        $InvoicesFromDate=$pluginData->$InvoicesFromDate;
```

Comment:
this is wrong, it would throw undefined variable, when accessing objects, do just this:
```suggestion
        $InvoicesFromDate=$pluginData->InvoicesFromDate;
```